src/core/mantle/evaluator/simple/expression/variableReplacer.ts

```diff
@@ -34,15 +34,15 @@ export function replaceVariables(expression: string, feature?: any): [string, JP
             return [result, []];
           }
         }
-        if (res.length == 1) {
+        if (res.length !== 0) {
```

Comment:
`res` length is not alway `1`, so I fixed condition.

Reply:
That's intended, Because As of now we cannot handle multiple output of a JSONPath evaluation.
And its suggested to only use JSONPath which gives single output for now.